### DISCUSSION TITLE
Cannot add remember.remember as verified domain

### DISCUSSION THREAD

**Discussion by ankursoni** [community/community#39834] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Cannot add remember.remember as verified domain as it throws an error:
"Domain is not a valid public domain."

**Comment by david-engelmann** [comment]
It seems like remember.remember domain is owned already. I guessing that Is you. Where did you register the domain?

**Reply by ankursoni** [reply]
Namecheap

**Comment by david-engelmann** [comment]
Did you just register it? Could it be related to [DNS Propagation](https://www.namecheap.com/support/knowledgebase/article.aspx/9622/10/dns-propagation-explained/)?

**Reply by ankursoni** [reply]
No, I registered weeks ago. This is a handshake TLD that am learning about now https://www.namecheap.com/support/knowledgebase/article.aspx/10484/2278/namecheap-handshake-tlds/
